src/components/action-popover/action-popover.spec.tsx

```diff
@@ -180,6 +181,421 @@ describe("ActionPopover", () => {
     );
   }
 
+  function renderDesignVariants(variant: number, props = {}, renderer = mount) {
```

Comment:
**comment**: by now I assume you know my feelings about this numeric `variant` prop in eamples 😅 Let's identify exactly which situations we're testing and create descriptive component/prop names so it's clear what exactly we're testing at each point.

Reply:
Yep, agreed 👍